test/create-element-to-jsx-gt-lt-entities.js

```diff
@@ -0,0 +1,3 @@
+var React = require('React');
+
+React.createElement('div', null, '\x3C\x3E');
```

Comment:
`\x3C\x3E` is how our CJSX compiler ends up outputting to ES5. I'm not sure if I should add support for other ways of writing these characters as well.